### PR TITLE
UPDATE config.go

### PR DESCRIPTION
**Pull request by guoguodan**
it may be null point

### CODE REVIEW

File: cmd/yurthub/app/config/config.go [hunk 1]
```diff
@@ -158,6 +158,10 @@ func Complete(options *options.YurtHubOptions) (*YurtHubConfiguration, error) {
 }
 
 func parseRemoteServers(serverAddr string) ([]*url.URL, error) {
+	if serverAddr == "" {
+		return make([]*url.URL, 0)
```
Comment (rambohe-ch) [inline cmd/yurthub/app/config/config.go:162]: how about change to `return []*url.URL{}, nil`?

Comment (rambohe-ch) [reply cmd/yurthub/app/config/config.go:162]: one return value(`error`) is missed.

Comment (guoguodan) [reply cmd/yurthub/app/config/config.go]: Thinks！ I've added one return value(error).

**Comment by rambohe-ch** [comment]
@guoguodan please merge these four commits into one commit.
image

**Comment by guoguodan** [comment]
thinks！ this pr close. I will create a new pr.

**Comment by rambohe-ch** [comment]
> thinks！ this pr close. I will create a new pr.

@guoguodan It's not need to create new pr, and you can git rebase these four commits into one commit, then git push --force  to overwrite the original branch.

**Pull request closed without merging**